be2-scala/src/main/scala/ch/epfl/pop/model/objects/Channel.scala

```diff
@@ -7,15 +7,15 @@ final case class Channel(channel: String) {
   /**
    * Extract the laoId from a channel name (even though it might be in the middle)
    *
-   * @return An Option of Base64Data corresponding to the decoded laoId or None if an error occurred
+   * @return A Hash corresponding to the decoded laoId or None if an error occurred
```

Comment:
Shouldn't we return an Option of Hash here if it fails? It seems a bit risky to me to have ??? in case of an error